### DISCUSSION TITLE
Upgrade MC on Balena Server

### DISCUSSION THREAD

**Discussion by MrEd83** [AlexProgrammerDE/balena-minecraft-server#159] [Q&A]
My server is currently running 1.16. What steps should I take to upgrade my server to 1.17?

Thanks for this!

**Comment by viedok** [comment]
You need to redeploy the project on balena then connect to your pi via ftp and replace the `paper.jar` file with the [latest paper jar](https://papermc.io/downloads).

**Reply by AlexProgrammerDE** [reply]
Alternatively you can also just set `ENABLE_UPDATE` to true after redeploying the project.

**Reply by stevereno30** [reply]
I know this is old, but I have enable_update set to true, but my server continues to run a very old paper.jar. Any idea why this might be? I've redeployed and nothing changes.

**Reply by viedok** [reply]
Yeah, I also haven't had a massive success with this feature. That's why I was recommending to replace it manually via FTP.
